src/Exception/RequestException.php

```diff
@@ -11,6 +11,9 @@
  */
 class RequestException extends TransferException
 {
+    /** @var int */
+    public static $responseSize = 120;
```

Comment:
missing phpdoc explaining what this does

Reply:
Maybe use a setter to enforce this being an integer.